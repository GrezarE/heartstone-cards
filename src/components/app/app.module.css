.app {
  width: 1440px;
  height: 100%;
  background-color: #D8CEB9;
  margin: auto;
}/*# sourceMappingURL=app.module.css.map */