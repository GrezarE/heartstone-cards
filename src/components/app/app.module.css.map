{"version":3,"sources":["app.module.scss","../../scss/variables.scss","app.module.css"],"names":[],"mappings":"AAEA;EACI,aAAA;EACA,YAAA;EACA,yBCKM;EDJN,YAAA;AEDJ","file":"app.module.css"}