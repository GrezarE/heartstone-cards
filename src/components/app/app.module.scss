@import '../../scss/variables.scss';

.app {
    width: 1440px;
    height: 100%;
    background-color: $vintage;
    margin: auto;
}