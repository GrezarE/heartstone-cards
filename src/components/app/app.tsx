import React from "react";
import { Header } from "../header/header";
import style from "./app.module.css";
// import { Routes, Route } from "react-router-dom";
import { Main } from "../main/main";

export const App = () => {
  return (
    <div className={style.app} data-testid="app">
      <Header />
      {/* <Routes> */}
      {/* <Route path="/" element={<Main />} />
      </Routes> */}
      <Main />
    </div>
  );
};
