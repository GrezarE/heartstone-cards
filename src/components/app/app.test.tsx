import { Provider } from "react-redux";
import { render, screen } from "@testing-library/react";
import { App } from "./app";
import configureMockStore from "redux-mock-store";
import thunk from "redux-thunk";

const mockStore = configureMockStore([thunk]);

describe("App", () => {
  it("app", () => {
    const store = mockStore({
      set: { set: null },
      selector: {
        className: "All",
        qualities: "All",
        setSelect: "",
      },
      loader: {
        loading: false,
      },
    });
    const app = render(
      <Provider store={store}>
        <App />
      </Provider>
    );

    const box = screen.getByTestId("app");
    expect(app).toMatchSnapshot();
    expect(box).toBeInTheDocument();
  });
});
