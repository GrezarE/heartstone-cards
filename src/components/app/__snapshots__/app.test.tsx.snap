// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App app 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="app"
        data-testid="app"
      >
        <header
          class="header"
        />
        <div
          class="main"
        >
          <div
            class="form"
          >
            <div
              class="box"
              data-testid="set-box"
            >
              <label
                data-testid="set-label"
                for="sets"
              >
                Sets:
              </label>
              <select
                data-testid="set-select"
                id="sets"
              >
                <option
                  value="Murder%20at%20Castle%20Nathria"
                >
                  Murder at Castle Nathria
                </option>
                <option
                  value="Forged%20in%20the%20Barrens"
                >
                  Forged in the Barrens
                </option>
                <option
                  value="United%20in%20Stormwind"
                >
                  United in Stormwind
                </option>
                <option
                  value="Fractured%20in%20Alterac%20Valley"
                >
                  Fractured in Alterac Valley
                </option>
                <option
                  value="Voyage%20to%20the%20Sunken%20City"
                >
                  Voyage to the Sunken City
                </option>
                <option
                  value="%22Core%22"
                >
                  Core
                </option>
              </select>
            </div>
            <div
              class="box"
              data-testid="class-box"
            >
              <label
                data-testid="class-label"
                for="sets"
              >
                Class:
              </label>
              <select
                data-testid="class-select"
                id="sets"
              >
                <option
                  value="All"
                >
                  All
                </option>
                <option
                  value="Demon Hunter"
                >
                  Demon Hunter
                </option>
                <option
                  value="Druid"
                >
                  Druid
                </option>
                <option
                  value="Hunter"
                >
                  Hunter
                </option>
                <option
                  value="Mage"
                >
                  Mage
                </option>
                <option
                  value="Paladin"
                >
                  Paladin
                </option>
                <option
                  value="Priest"
                >
                  Priest
                </option>
                <option
                  value="Rogue"
                >
                  Rogue
                </option>
                <option
                  value="Shaman"
                >
                  Shaman
                </option>
                <option
                  value="Warlock"
                >
                  Warlock
                </option>
                <option
                  value="Warrior"
                >
                  Warrior
                </option>
                <option
                  value="Neutral"
                >
                  Neutral
                </option>
              </select>
            </div>
            <div
              class="box"
            >
              <label
                for="sets"
              >
                Qualities:
              </label>
              <select
                id="sets"
              >
                <option
                  value="All"
                >
                  All
                </option>
                <option
                  value="Common"
                >
                  Common
                </option>
                <option
                  value="Rare"
                >
                  Rare
                </option>
                <option
                  value="Epic"
                >
                  Epic
                </option>
                <option
                  value="Legendary"
                >
                  Legendary
                </option>
              </select>
            </div>
          </div>
          <ul />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="app"
      data-testid="app"
    >
      <header
        class="header"
      />
      <div
        class="main"
      >
        <div
          class="form"
        >
          <div
            class="box"
            data-testid="set-box"
          >
            <label
              data-testid="set-label"
              for="sets"
            >
              Sets:
            </label>
            <select
              data-testid="set-select"
              id="sets"
            >
              <option
                value="Murder%20at%20Castle%20Nathria"
              >
                Murder at Castle Nathria
              </option>
              <option
                value="Forged%20in%20the%20Barrens"
              >
                Forged in the Barrens
              </option>
              <option
                value="United%20in%20Stormwind"
              >
                United in Stormwind
              </option>
              <option
                value="Fractured%20in%20Alterac%20Valley"
              >
                Fractured in Alterac Valley
              </option>
              <option
                value="Voyage%20to%20the%20Sunken%20City"
              >
                Voyage to the Sunken City
              </option>
              <option
                value="%22Core%22"
              >
                Core
              </option>
            </select>
          </div>
          <div
            class="box"
            data-testid="class-box"
          >
            <label
              data-testid="class-label"
              for="sets"
            >
              Class:
            </label>
            <select
              data-testid="class-select"
              id="sets"
            >
              <option
                value="All"
              >
                All
              </option>
              <option
                value="Demon Hunter"
              >
                Demon Hunter
              </option>
              <option
                value="Druid"
              >
                Druid
              </option>
              <option
                value="Hunter"
              >
                Hunter
              </option>
              <option
                value="Mage"
              >
                Mage
              </option>
              <option
                value="Paladin"
              >
                Paladin
              </option>
              <option
                value="Priest"
              >
                Priest
              </option>
              <option
                value="Rogue"
              >
                Rogue
              </option>
              <option
                value="Shaman"
              >
                Shaman
              </option>
              <option
                value="Warlock"
              >
                Warlock
              </option>
              <option
                value="Warrior"
              >
                Warrior
              </option>
              <option
                value="Neutral"
              >
                Neutral
              </option>
            </select>
          </div>
          <div
            class="box"
          >
            <label
              for="sets"
            >
              Qualities:
            </label>
            <select
              id="sets"
            >
              <option
                value="All"
              >
                All
              </option>
              <option
                value="Common"
              >
                Common
              </option>
              <option
                value="Rare"
              >
                Rare
              </option>
              <option
                value="Epic"
              >
                Epic
              </option>
              <option
                value="Legendary"
              >
                Legendary
              </option>
            </select>
          </div>
        </div>
        <ul />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
